specs/notation-config.md

```diff
@@ -0,0 +1,59 @@
+# Notation Configuration
+
+To enable persisted configuration, simplifying the execution of the `notation` cli, the following configuration file will be available
+
+> Note: there will be a policy based configuration that will come at a later point.
+
+## Location
+
+The default location and file will be stored at: `~/.notary/notation-config.json`. The `notation` cli and libraries will support alternate locations through a `config-location` parameters.
+
+## Properties
+
+- `enabled` - bool, which acts as an on/off switch for default notation behavior. --enabled false would disable all automatic validations
+- `verificationCerts` - collection of name/value pairs for a collection of public certs that are used for verification. These may be replaced with a future policy configuration.
+  - `name` - a named reference to the certificate
+  - `path` - a location by which the certificate can be found by the notation cli or notation libraries
+- `signing-keys` - a collection of name/value pairs of signing keys.
+  - `name` - a named reference to the key
+  - `path` - a location by which the key can be found by the notation cli or notation libraries
+  - `default` - the signing key to be used when `notation sign` is called without `--name`
+- `insecureRegistries` - a list of registries that may be used without https
```

Comment:
Should `insecureRegistries` be called `plainHTTP`?

Reply:
For better or worse, I was copying the docker config. which is what we have in prototype-2. 
I'm going to leave for now. If folks want to change, can we open an issue to discuss?
```json
{
  "insecure-registries": [
    "registry.wabbit-networks.io",
    "localhost"
  ]
}
```